2021-02-28 03:51:18.487 - INFO - 20290 - azureml.train.automl.automlconfig._start_execution:211 - Submitting remote.
2021-02-28 04:15:13.561 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:15:13.572 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:15:13.595 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:15:13.746 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=151.47[ms]
2021-02-28 04:15:13.746 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=174.76[ms]
2021-02-28 04:15:13.756 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:15:18.751 - INFO - 20290 - azureml.automl.runtime.distributed.utilities.horovod_initialized:58 - Horovod not found in current environment. Distribution via Horovod will be disabled.
2021-02-28 04:15:23.130 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:15:23.130 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:15:25.611 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:15:25.764 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:15:25.765 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=12008.64[ms]
2021-02-28 04:15:25.775 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:15:25.775 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=12214.24[ms]
2021-02-28 04:16:02.030 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:16:02.036 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:16:02.057 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:16:02.200 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=142.76[ms]
2021-02-28 04:16:02.200 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=164.0[ms]
2021-02-28 04:16:02.201 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:16:02.789 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:16:02.789 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:16:02.805 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:16:02.906 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:16:02.906 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=705.07[ms]
2021-02-28 04:16:02.908 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:16:02.908 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=878.07[ms]
2021-02-28 04:16:29.731 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:16:29.737 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:16:29.758 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:16:29.913 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=154.17[ms]
2021-02-28 04:16:29.913 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=175.56[ms]
2021-02-28 04:16:29.914 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:16:30.495 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:16:30.495 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:16:30.512 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:16:30.606 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:16:30.606 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=692.57[ms]
2021-02-28 04:16:30.608 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:16:30.608 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=876.97[ms]
2021-02-28 04:17:02.766 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:17:02.780 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:17:02.784 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:17:02.925 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=140.76[ms]
2021-02-28 04:17:02.925 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=144.87[ms]
2021-02-28 04:17:02.927 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:17:03.551 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:17:03.552 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:17:03.570 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:17:03.672 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:17:03.672 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=745.39[ms]
2021-02-28 04:17:03.674 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:17:03.674 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=907.77[ms]
2021-02-28 04:17:47.020 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:17:47.035 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:17:47.048 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:17:47.187 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=139.05[ms]
2021-02-28 04:17:47.187 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=152.38[ms]
2021-02-28 04:17:47.188 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:17:47.658 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:17:47.658 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:17:47.675 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:17:47.758 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:17:47.758 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=570.34[ms]
2021-02-28 04:17:47.760 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:17:47.760 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=740.04[ms]
2021-02-28 04:18:22.792 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:18:22.799 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:18:22.819 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:18:22.962 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=143.11[ms]
2021-02-28 04:18:22.962 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=163.6[ms]
2021-02-28 04:18:22.963 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:18:23.482 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:18:23.482 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:18:23.499 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:18:23.734 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:18:23.734 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=771.38[ms]
2021-02-28 04:18:23.736 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:18:23.737 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=944.4[ms]
2021-02-28 04:19:48.446 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:19:48.450 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:19:48.466 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:19:48.610 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=143.71[ms]
2021-02-28 04:19:48.610 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=159.98[ms]
2021-02-28 04:19:48.611 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:19:49.081 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:19:49.081 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:19:49.098 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:19:49.322 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:19:49.323 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=711.84[ms]
2021-02-28 04:19:49.324 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:19:49.325 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=878.84[ms]
2021-02-28 04:22:16.637 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:22:16.642 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:22:16.664 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:22:16.792 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=128.42[ms]
2021-02-28 04:22:16.792 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=150.17[ms]
2021-02-28 04:22:16.793 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:22:17.397 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:22:17.397 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:22:17.420 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:22:17.565 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:22:17.566 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=772.31[ms]
2021-02-28 04:22:17.567 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:22:17.567 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=930.1[ms]
2021-02-28 04:23:39.732 - INFO - 20290 - azureml.train.automl.automlconfig._start_execution:211 - Submitting remote.
2021-02-28 04:45:48.757 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:45:48.791 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 04:45:48.810 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 04:45:48.949 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=139.11[ms]
2021-02-28 04:45:48.949 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=157.71[ms]
2021-02-28 04:45:48.958 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 04:45:49.484 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 04:45:49.484 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 04:45:49.486 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:45:49.566 - CRITICAL - 20290 - azureml.train.automl._model_download_utilities.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:45:49.567 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Failure, Duration=608.68[ms]
2021-02-28 04:45:49.568 - CRITICAL - 20290 - azureml.train.automl.run.log_traceback:224 - Type: Unclassified
Class: AttributeError
Message: /anaconda/envs/azureml_py36/lib/libxgboost.so: undefined symbol: XGBoosterUnserializeFromBuffer
Traceback:
  File "telemetry_activity_logger.py", line 57, in _log_activity
    yield
  File "run.py", line 701, in get_output
    fitted_model = _download_automl_model(curr_run, model_name)
  File "_model_download_utilities.py", line 90, in _download_automl_model
    model = _load_automl_model(model_path)
  File "_model_download_utilities.py", line 174, in _load_automl_model
    fitted_model = pickle.load(model_file)  # type: Optional[Any]
  File "core.py", line 1088, in __setstate__
    _LIB.XGBoosterUnserializeFromBuffer(handle, ptr, length))
  File "__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))

ExceptionTarget: Unspecified
2021-02-28 04:45:49.568 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Failure, Duration=811.07[ms]
2021-02-28 04:50:57.117 - INFO - 20290 - azureml.train.automl.automlconfig._start_execution:211 - Submitting remote.
2021-02-28 05:19:49.181 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:19:49.214 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:19:49.223 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 05:19:49.791 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=568.22[ms]
2021-02-28 05:19:49.792 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=577.15[ms]
2021-02-28 05:19:49.802 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 05:19:50.896 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 05:19:50.897 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 05:19:50.991 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Success, Duration=1189.18[ms]
2021-02-28 05:19:50.992 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=1810.81[ms]
2021-02-28 05:24:30.442 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:24:30.461 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:24:30.479 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 05:24:30.614 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=134.57[ms]
2021-02-28 05:24:30.614 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=152.34[ms]
2021-02-28 05:24:30.642 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 05:24:31.000 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 05:24:31.000 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 05:24:31.017 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Success, Duration=374.78[ms]
2021-02-28 05:24:31.017 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=575.3[ms]
2021-02-28 05:26:11.056 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:26:11.088 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:26:11.103 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 05:26:11.925 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=822.2[ms]
2021-02-28 05:26:11.925 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=836.91[ms]
2021-02-28 05:26:11.948 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 05:26:14.068 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 05:26:14.068 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 05:26:14.089 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Success, Duration=2140.61[ms]
2021-02-28 05:26:14.089 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=3032.87[ms]
2021-02-28 05:52:45.182 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:52:45.189 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 05:52:45.211 - INFO - 20290 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 05:52:45.640 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=428.88[ms]
2021-02-28 05:52:45.640 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=451.48[ms]
2021-02-28 05:52:45.641 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 05:52:46.246 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 05:52:46.246 - INFO - 20290 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 05:52:46.264 - INFO - 20290 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Success, Duration=622.06[ms]
2021-02-28 05:52:46.264 - INFO - 20290 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=1082.29[ms]
2021-02-28 18:13:52.693 - INFO - 8492 - azureml.train.automl.automlconfig._start_execution:211 - Submitting remote.
2021-02-28 18:43:21.505 - INFO - 8492 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 18:43:21.516 - INFO - 8492 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetOutput
2021-02-28 18:43:21.532 - INFO - 8492 - azureml.train.automl.run._log_activity:54 - ActivityStarted: GetChildren
2021-02-28 18:43:21.691 - INFO - 8492 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetChildren, HowEnded=Success, Duration=159.21[ms]
2021-02-28 18:43:21.691 - INFO - 8492 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=175.18[ms]
2021-02-28 18:43:21.700 - INFO - 8492 - azureml.train.automl._model_download_utilities._log_activity:54 - ActivityStarted: DownloadModel
2021-02-28 18:43:23.352 - INFO - 8492 - azureml.automl.runtime.distributed.utilities.horovod_initialized:58 - Horovod not found in current environment. Distribution via Horovod will be disabled.
2021-02-28 18:43:24.179 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer.__init__:174 - Feature sweeping enabled: False
2021-02-28 18:43:24.179 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer.__init__:175 - Feature sweeping timeout: 86400
2021-02-28 18:43:24.460 - INFO - 8492 - azureml.train.automl._model_download_utilities._log_activity:92 - ActivityCompleted: Activity=DownloadModel, HowEnded=Success, Duration=2760.24[ms]
2021-02-28 18:43:24.460 - INFO - 8492 - azureml.train.automl.run._log_activity:92 - ActivityCompleted: Activity=GetOutput, HowEnded=Success, Duration=2955.11[ms]
2021-02-28 18:57:45.837 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:491 - Starting to apply transforms. Total ram: 14706556928 bytes
2021-02-28 18:57:45.838 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:45.838 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.858 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 45)
2021-02-28 18:57:45.858 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.858 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - CatImputer,StringCastTransformer,LabelEncoderTransformer: Applying transform.
2021-02-28 18:57:45.866 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - CatImputer,StringCastTransformer,LabelEncoderTransformer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:45.866 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.866 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.874 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 6)
2021-02-28 18:57:45.885 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.885 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.892 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 7)
2021-02-28 18:57:45.893 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.893 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.900 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 4)
2021-02-28 18:57:45.900 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.900 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.907 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 6)
2021-02-28 18:57:45.908 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.908 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.915 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 4)
2021-02-28 18:57:45.915 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.915 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.925 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 44)
2021-02-28 18:57:45.925 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.926 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.942 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 6)
2021-02-28 18:57:45.957 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.957 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.967 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 7)
2021-02-28 18:57:45.967 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.967 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - StringCastTransformer,CountVectorizer: Applying transform.
2021-02-28 18:57:45.977 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - StringCastTransformer,CountVectorizer: Finished applying transform. Shape (2069, 7)
2021-02-28 18:57:45.985 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4297027584 bytes
2021-02-28 18:57:45.986 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:45.988 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:45.990 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:45.990 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:45.992 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:45.993 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:45.993 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:45.995 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:45.996 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:45.996 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:45.999 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:45.999 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:46.000 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:46.002 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:46.002 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:46.002 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:46.004 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:46.005 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:46.005 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:46.007 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:46.007 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:502 - Transform memory estimate: 0 bytes, Current available memory: 4296994816 bytes
2021-02-28 18:57:46.008 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:512 - SimpleImputer: Applying transform.
2021-02-28 18:57:46.010 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:526 - SimpleImputer: Finished applying transform. Shape (2069, 1)
2021-02-28 18:57:46.010 - INFO - 8492 - azureml.automl.runtime.featurization.data_transformer._apply_transforms:535 - Finished applying transforms
2021-02-28 18:57:46.010 - INFO - 8492 - azureml.automl.runtime.shared.memory_utilities._is_low_memory:208 - Total ram: 14706556928 bytes, Current Available memory: 4296994816 bytes. Low memory threshold: 0.5
